portfolio/src/main/webapp/script.js

```diff
@@ -30,10 +30,22 @@ const addRandomFact = async () => {
     factContainer.innerText =
         'Sorry, an error occured. Please try again later.';
   }
-}
+};
 
-const addComments = async () => {
+/** Adds a new comment to the DOM. */
+const addComment = (comment) => {
```

Comment:
You might want to consider moving all the comments logic to its own class. Paul can help you do this if you aren't sure how.

Reply:
Moved all comment logic to its own module.